website/manage_and_send_mail.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+
+import contextlib
+import io
+import os
+import sys
+import traceback
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "website.settings")
+os.environ.setdefault('DJANGO_CONFIGURATION', 'Dev')
```

Comment:
```suggestion
os.environ.setdefault("DJANGO_CONFIGURATION", "Dev")
```

Para mantener la consistencia de los `"`

Reply:
Es una buena idea, pero preferí dejarlo igual, para que si uno hace un diff contra `manage.py` no tenga esas diferencias.